### ISSUE TITLE
False negative "use of moved value" error after colling consuming method twice

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#5093]
## Environment

* **IntelliJ Rust plugin version:** 0.2.119.3012-201-nightly
* **Rust toolchain version:** 1.41.0 (5e1a79984 2020-01-27) x86_64-apple-darwin
* **IDE name and version:** CLion 2020.1 EAP (CL-201.6251.14)
* **Operating system:** macOS 10.15.3

## Problem description

I expect plugin to report "use of moved value" error when I call a method accepting `self` twice, but there is no warning.

## Steps to reproduce

```rust
struct User {
    name: String
}

impl User {
    fn die(self) {
        println!("dead {}", self.name);
    }
}

fn main() {
    let user = User { name: String::from("Someone") };
    user.die();
    user.die();
}
```
